W-18543667 added new Generating API Documentation with Einstein jh (#305)
mulesoft/docs-exchange: modules/ROOT/pages/workflow-sharing-assets.adoc

Anypoint Exchange provides the ability to publish assets in private Exchange instances and Exchange public portals. Publishing is the process of creating assets and making them available to consumers for discovery and reuse.

. xref:to-create-an-asset.adoc[Create an Asset] - Create new assets in Exchange.

> . xref:to-describe-an-asset.adoc[Edit an Asset] - Edit asset information like the description and documentation, add additional pages, terms and conditions, and tags, and change versions.

Reviewer: ```suggestion
. xref:to-describe-an-asset.adoc[Edit an Asset] - Edit asset information such as the description and documentation. You can also change versions and add pages, terms and conditions, and tags.
```
